onert-micro/onert-micro/include/pal/mcu/PALReduceCommon.h

```diff
@@ -37,6 +37,8 @@ namespace pal
 template <class T>
 struct ReduceSumFn
 {
+  constexpr static T InitValue = T(0);
```

Comment:
(Optional) In other parts of `ONE` and `onert-micro` `static` members are written in lower case, like `initValue` or `kInitValue`.